packages/ui/src/components/calendar/DatePresenter.svelte

```diff
@@ -22,10 +22,14 @@
   import Label from '../Label.svelte'
   import DPCalendar from './icons/DPCalendar.svelte'
   import DPCalendarOver from './icons/DPCalendarOver.svelte'
-  import { getMonthName } from './internal/DateUtils'
+  import { getMonthName, getWorkDate, getWorkDayType } from './internal/DateUtils'
```

Comment:
Better to create a separate component in tracker and not burden this one with extra functionality.

Reply:
Done.